data/images-scraped.ts

```diff
@@ -19,11 +19,11 @@ export const scrapedImages: ScrapedImages[] = [
     author: authors.aikoyori,
     className: 'object-contain'
   },
-  {
-    ownerRepoPath: 'SAWARATSUKI/ServiceLogos',
-    author: authors.sawaratsuki,
-    className: 'object-contain'
-  },
+  // {
+  //   ownerRepoPath: 'SAWARATSUKI/ServiceLogos',
+  //   author: authors.sawaratsuki,
+  //   className: 'object-contain'
+  // },
```

Comment:
I had to temporarily disable it as the repo is private right now, and it was causing CI to be sad. Hopefully it can come back soon!